src/zwm.zig

```diff
@@ -305,18 +307,18 @@ fn manageWindow(window: c.Window, window_attrs: c.XWindowAttributes, display: *c
     { // client update title
         var tp: c.XTextProperty = undefined;
         defer _ = c.XFree(tp.value);
-        var AtomNetWMName = c.XInternAtom(display, "_NET_WM_NAME", c.False);
+        const AtomNetWMName = c.XInternAtom(display, "_NET_WM_NAME", c.False);
         if (c.XGetTextProperty(display, window, &tp, AtomNetWMName) == 0) {
             print("Title of client could not be read from EWMH\n", .{});
-            var XA_WM_NAME: c.Atom = 39;
+            const XA_WM_NAME: c.Atom = 39;
             if (c.XGetTextProperty(display, window, &tp, XA_WM_NAME) != 0)
                 print("Title of client  could not be read from ICCCM\n", .{});
         }
         if (tp.nitems == 0)
-            std.mem.copy(u8, cl.name[0..25], "NAME UNKNOWN");
+            std.mem.copyForwards(u8, cl.name[0..25], "NAME UNKNOWN");
```

Comment:
I think we should use `@memcpy` instead. 
https://www.openmymind.net/Zigs-memcpy-copyForwards-and-copyBackwards/